javascript-practice/src/views/components/product-item.html

```diff
@@ -0,0 +1,27 @@
+<div class="product-item">
+  <figure class="product-thumbnail">
+    <a href="javascript:void(0)">
```

Comment:
About UI/UX. You should group a product into a `<a>` tag. When users hover on the product, they can click to redirect to the detail page instead of only clicking on each thumbnail